go.mod

```diff
@@ -16,6 +16,7 @@ require (
 	github.com/hashicorp/hc-install v0.9.0 // MPL 2.0
 	github.com/hashicorp/terraform-exec v0.21.0 // MPL 2.0
 	github.com/hashicorp/terraform-json v0.23.0 // MPL 2.0
+	github.com/hexops/gotextdiff v1.0.3 // Golang license
```

Comment:
What kind of license is this?

We need to make sure it is compatible/importable.

Reply:
BSD 3-Clause "New" or "Revised" License https://github.com/hexops/gotextdiff/blob/main/LICENSE